### ISSUE TITLE
no-unused-vars in ES6 names imports

### ISSUE THREAD

**Issue by Jangla** [babel/babel-eslint#345]
Some named member imports in ES6 modules are flagged as unused vars, even when they're used.

In this example, List and Map work fine. toJS, however gets flagged even though it's used.

In Atom: http://puu.sh/q9jl2/e23d6e733c.png

.eslintrc:

`
"env": {
  "browser": true,
  "es6": true
},
"extends": "eslint:recommended",
"installedESLint": true,
"parser": "babel-eslint",
"parserOptions": {
  "sourceType": "module",
  "ecmaVersion": 6,
  "ecmaFeatures": {
    "jsx": true,
    "modules": true,
    "experimentalObjectRestSpread": true
  },
  "sourceType": "module"
},
"plugins": [
  "react", "babel", "import"
],
"rules": {
  "no-class-assign": "off",
  "no-unused-vars": "warn",
  "react/jsx-uses-react": "error",
  "react/jsx-uses-vars": "error",
  "import/no-unresolved": "error",
  "import/prefer-default-export": "warn",
  "indent": [
    "warn",
    "tab",
    { "SwitchCase": 1, "VariableDeclarator": 1 }
  ],
...
`

**Comment by danez** [comment]
I don't see the imported `toJS` and `fromJS` being used in the screenshot. I see `state.get(...).toJS()` at the bottom, but this is not the imported function.

**Comment by Jangla** [comment]
screenshot updated to show the eslint warning. 

And `state.get('accounts').toJS();` definitely is the imported function.

**Comment by danez** [comment]
I'm sorry, but it is not. If you would like to use the imported function you would have to do this:
```js
import { toJS } from '';

toJS(something);
```

The function you are using is a function that is part of the result object from `state.get()`.
To do `state.get().toJS()` you don't need an import for `toJS`.

**Comment by Jangla** [comment]
![facepalm](https://cloud.githubusercontent.com/assets/996667/17025513/161e3cfe-4f54-11e6-80e6-03e8816befc8.jpg)


**facepalm**

**Comment by danez** [comment]
No problem, that happened to all of us already :)
